cli/engine/ApplyEngine.py

```diff
@@ -99,6 +97,14 @@ def load_manifest(self):
         if os.path.isfile(path_to_manifest):
             self.manifest_docs = load_manifest(get_build_path(self.cluster_model.specification.name))
 
+    def assert_incompatible_terraform(self):
```

Comment:
Maybe this method name can be changed to something like check_if_incompatible_terraform_version?

Reply:
We want to assert not just check so leaving as is.